src/Network/Probe.php

```diff
@@ -1063,7 +1063,8 @@ private function diaspora($webfinger) {
 	private function ostatus($webfinger) {
 		$data = array();
 		if (is_array($webfinger["aliases"])) {
-			foreach ($webfinger["aliases"] as $alias) {
```

Comment:
This "as" is replaced with a "AS" and a comment saying it should be a lowercase "as"